plugins/frontend/backstage-plugin-cloudsmith/src/components/CloudsmithQuotaCard/Content.tsx

```diff
@@ -0,0 +1,127 @@
+/*
+ * Copyright 2022 Larder Software Limited
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import React from 'react';
+import { CloudsmithQuotaCardContentProps } from './types';
+import { CloudsmithClient } from '../../api';
+import {
+  discoveryApiRef,
+  fetchApiRef,
+  useApi,
+} from '@backstage/core-plugin-api';
+import useAsync from 'react-use/lib/useAsync';
+import { ErrorPanel, Progress } from '@backstage/core-components';
+import {
+  Paper,
+  Table,
+  TableCell,
+  TableContainer,
+  TableHead,
+  TableRow,
+  TableBody,
+  Card,
+  CardHeader,
+  Avatar,
+} from '@material-ui/core';
+
+/**
+ * A component to render the stats about a cloudsmith usage quota.
+ *
+ * @public
+ */
+export const Content = ({ owner }: CloudsmithQuotaCardContentProps) => {
```

Comment:
It might be nice to use a [Gauge](https://backstage.io/storybook/?path=/story/data-display-gauge--static-color) or a [linear gauge](https://backstage.io/storybook/?path=/story/data-display-lineargauge--low-progress) for this info?